RealmSwift/RealmCollection.swift

```diff
@@ -223,7 +239,16 @@ extension AnyRealmValue: RealmCollectionValue {
     }
 }
 
-extension Optional: RealmCollectionValue where Wrapped: RealmCollectionValue {
+extension Optional: RealmCollectionValue where Wrapped: RealmCollectionValue,
+                                               Wrapped: _DefaultConstructible {
+    /// :nodoc:
+    public static func _rlmDefaultValue(_ forceDefaultInstantiation: Bool) -> Optional<Wrapped> {
```

Comment:
For `_rlmDefaultValue` in general I would consider changing the parameter name from `forceDefaultInstantiation` to `forceDefaultConstruction` or `forceDefaultInitialization` so that we have _some_ consistency.

Reply:
changed to forceDefaultInitialization